src/factom-event-emitter.js

```diff
@@ -43,9 +48,22 @@ Object.freeze(FACTOM_EVENT);
  * emitter.on('FA29eyMVJaZ2tbGqJ3M49gANaXMXCjgfKcJGe5mx8p4iQFCvFDAC', (transaction) => ...);
  * // Listen to any new entries in a given chain
  * emitter.on('4060c0192a421ca121ffff935889ef55a64574a6ef0e69b2b4f8a0ab919b2ca4', (entryBlock) => ...);
+ * // Listen to any pending transactions involving a given Factoid address
+ * emitter.on(FactomEventEmitter.getSubscriptionToken({
+ *  eventType: 'newPendingTransaction', address: 'FA29eyMVJaZ2tbGqJ3M49gANaXMXCjgfKcJGe5mx8p4iQFCvFDAC'
+ * }), (pendingTransaction) => ...);
  */
 class FactomEventEmitter extends EventEmitter {
-    constructor(cli, opts) {
+    /**
+     * Given an event configuration object returns a tokenized string
+     * @param {object} event
```

Comment:
Can you document the object properties? https://jsdoc.app/tags-param.html#parameters-with-properties